Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1297,10 +1297,12 @@ public extension Purchases {
 
     /**
      * Use this function to open the manage subscriptions page.
-
-     * - Parameter completion: A completion block that is called when the modal is closed.
-     * If it was not successful, there will be an `Error`.
-
+     *
+     * - Parameter completion: A completion block that will be called when the modal is opened,
+     * not when it's actually closed. ß∫This is because of an undocumented change in StoreKit's behavior
```

Comment:
```suggestion
     * not when it's actually closed. This is because of an undocumented change in StoreKit's behavior
```

Reply:
great catch 😅 I'm not even sure how I managed to add that in there